test/e2e/set/scyllacluster/scyllacluster_tls.go

```diff
@@ -260,6 +265,130 @@ var _ = g.Describe("ScyllaCluster", func() {
 		}
 	})
 
+	g.It("should rotate TLS certificate without disrupting ongoing CQL traffic", func() {
+		if !utilfeature.DefaultMutableFeatureGate.Enabled(features.AutomaticTLSCertificates) {
+			g.Skip(fmt.Sprintf("Skipping because %q feature is disabled", features.AutomaticTLSCertificates))
+		}
+
+		ctx, cancel := context.WithTimeout(context.Background(), testTimeout)
+		defer cancel()
+
+		sc := f.GetDefaultScyllaCluster()
+		o.Expect(sc.Spec.Datacenter.Racks).To(o.HaveLen(1))
```

Comment:
we know what the manifest is and we don't assert this in other tests, I'd skip it

Reply:
This test only relies on fact of having single node cluster, hence it's asserted.